tools/pio/pre_custom_esp32.py

```diff
@@ -66,6 +66,7 @@
     "-DUSES_CHART_JS",
 
     "-DUSE_SETTINGS_ARCHIVE",
+    "-DFEATURE_ESP_P2P",
```

Comment:
This should be enabled by default to be backward compatible with current feature set, and be undefined at will. The regular builds should still include the P2P feature.

Maybe it would be better to change this from a `bool` to an `integer` define, like this:
```
#ifndef FEATURE_ESP_P2P
#define FEATURE_ESP_P2P 1  // Defaults to on
#endif // ifndef FEATURE_ESP_P2P
```
In your special builds or `Custom.h` you could use:
```
#if defined(FEATURE_ESP_P2P) && FEATURE_ESP_P2P
#undef FEATURE_ESP_P2P
#define FEATURE_ESP_P2P 0  // Disable P2P feature
#endif // if defined(FEATURE_ESP_P2P) && FEATURE_ESP_P2P
```
And then test the setting with:
```
#if FEATURE_ESP_P2P // Enabled?
...
```
To turn it off in the pre_custom* Python scripts:
```
"-DFEATURE_ESP_P2P=0"
```